src/pages/hardhead/awards/election/stallone.tsx

```
@@ -0,0 +1,182 @@
+import Post from "../../../../components/post";
+import { useAuth } from "../../../../context/auth";
+import { isMobile } from "react-device-detect";
+import { ElectionModuleProps } from ".";
+import { useLocation, useNavigate } from "react-router-dom";
+import { Nomination } from "../../../../types/nomination";
+
+const Stallone = ({
+  ID,
+  Name,
+  Description,
+  Date,
```

Biome (2.1.2)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)